### PR TITLE
feat(uikit): Prediction icons

### PR DESCRIPTION
**Pull request by hachiojidev**
Adds 2 icons used in predictions

### CODE REVIEW

**Comment by Chef-Cheems** [comment]
@hachiojidev what about [CardsIcon](https://github.com/pancakeswap/pancake-frontend/pull/602/files#diff-9fceeb55c445bf6e1279d0deb0b851076b082bb3ceb540793b2e5396316526a2R1) and [2 multiplier icons](https://github.com/pancakeswap/pancake-frontend/pull/602/files#diff-12cbf0831bec8f8520ef77ca3dbe0ad7f5dba214e2b077776390a51cce84a14aR14)?

File: packages/pancake-uikit/src/components/Svg/Icons/BunnyCards.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          />
+          <path d="M30.936 51.698L26.3172 54.3647L39.4698 31.5838L41.0094 34.2505L30.936 51.698Z" fill="white" />
+          <path
+            d="M24.0449 10.2847L7.1336 39.5759L5.59399 36.9092L20.1155 11.7572L21.4272 10.9999C22.2535 10.5229 23.1544 10.292 24.0449 10.2847Z"
+            fill="white"
+          />
+        </g>
+      </g>
+      <defs>
+        <clipPath id="clip0">
```
Comment (Chef-Cheems) [inline packages/pancake-uikit/src/components/Svg/Icons/BunnyCards.tsx:72]: Better to change this to `clip-bunnycards` and same on line 8, or there'll be a risk to face similar issues on Safari with icons clipping cause the clip id is the same.

Comment (hachiojidev) [reply packages/pancake-uikit/src/components/Svg/Icons/BunnyCards.tsx:72]: Ah damn, you're right.

**Comment by hachiojidev** [comment]
@Chef-Cheems CardsIcon is already in the UI Kit as Cards. Name needs to be updated. I am keeping the multiplier icons out of the UI Kit for now because they are very specific to predictions and they are not traditional icons.

**Review by Chef-Cheems — APPROVED** [review]
LGTM 👍

**Pull request merged**